helpers/errors.helpers.js

```diff
@@ -0,0 +1,22 @@
+const catchError = (err, res) => {
```

Comment:
We have a catchError function, but isn't used anywhere?

Reply:
This function is not in use, removed for now .